portfolio/src/main/webapp/index.html

```diff
@@ -10,6 +10,7 @@
     <script src="script.js"></script>
     <script src="indexScript.js"></script>
     <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=INSERT_KEY_HERE"></script>
```

Comment:
After talking with Iris, I've come to the conclusion that there's no way to win, so in this case it's okay to put your API key here.

Since it's being used from JS, it's impossible to avoid exposing your API key on your website since it must be in the user's browser for JS to use it. (In other cases, an API key might only be used by the server which is not user-exposed, so it would then make sense not to check it in to a GitHub repo because that would newly expose it, unlike here.) Since your API key is exposed, might as well expose it even more for convenience here? At least that's the argument.